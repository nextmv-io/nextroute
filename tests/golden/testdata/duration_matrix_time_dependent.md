# Time dependent duration matrix (duration_matrix_time_dependent*.json)
