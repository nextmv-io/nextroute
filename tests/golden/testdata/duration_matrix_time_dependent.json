{
  "duration_matrix": {
    "default_matrix": [
      [0, 720, 1020, 0, 0],
      [720, 0, 660, 0, 0],
      [1020, 660, 0, 0, 0],
      [0, 0, 0, 0, 0],
      [0, 0, 0, 0, 0]
    ],
    "time_frames": [
      {
        "start_time": "2023-01-01T12:03:00Z",
        "end_time": "2023-01-01T12:10:00Z",
        "matrix": [
          [0, 800, 1100, 0, 0],
          [800, 0, 740, 0, 0],
          [1100, 740, 0, 0, 0],
          [0, 0, 0, 0, 0],
          [0, 0, 0, 0, 0]
        ]
      },
      {
        "start_time": "2023-01-01T12:12:00Z",
        "end_time": "2023-01-01T12:20:00Z",
        "matrix": [
          [0, 850, 1120, 0, 0],
          [850, 0, 720, 0, 0],
          [1120, 720, 0, 0, 0],
          [0, 0, 0, 0, 0],
          [0, 0, 0, 0, 0]
        ]
      }
    ]
  },
  "stops": [
    {
      "id": "Fushimi Inari Taisha",
      "location": { "lon": 135.772695, "lat": 34.967146 }
    },
    {
      "id": "Kiyomizu-dera",
      "location": { "lon": 135.78506, "lat": 34.994857 }
    },
    {
      "id": "Nij≈ç Castle",
      "location": { "lon": 135.748134, "lat": 35.014239 }
    }
  ],
  "vehicles": [
    {
      "id": "v1",
      "start_time": "2023-01-01T12:00:00Z"
    }
  ]
}
